crates/bevy_input/src/keyboard.rs

```diff
@@ -31,206 +46,365 @@ pub fn keyboard_input_system(
     Key6,
-    /// The '7' key over the letters.
+    /// The `7` key over the letters.
     Key7,
-    /// The '8' key over the letters.
+    /// The `8` key over the letters.
     Key8,
-    /// The '9' key over the letters.
+    /// The `9` key over the letters.
     Key9,
-    /// The '0' key over the 'O' and 'P' keys.
+    /// The `0` key over the `O` and `P` keys.
     Key0,
 
+    /// The `A` key.
     A,
+    /// The `B` key.
     B,
+    /// The `C` key.
     C,
+    /// The `D` key.
     D,
+    /// The `E` key.
     E,
+    /// The `F` key.
     F,
+    /// The `G` key.
     G,
+    /// The `H` key.
     H,
+    /// The `I` key.
     I,
+    /// The `J` key.
     J,
+    /// The `K` key.
     K,
+    /// The `L` key.
     L,
+    /// The `M` key.
     M,
+    /// The `N` key.
     N,
+    /// The `O` key.
     O,
+    /// The `P` key.
     P,
+    /// The `Q` key.
     Q,
+    /// The `R` key.
     R,
+    /// The `S` key.
     S,
+    /// The `T` key.
     T,
+    /// The `U` key.
     U,
+    /// The `V` key.
     V,
+    /// The `W` key.
     W,
+    /// The `X` key.
     X,
+    /// The `Y` key.
     Y,
+    /// The `Z` key.
     Z,
 
-    /// The Escape key, next to F1.
+    /// The `Escape` / `ESC` key, next to the `F1` key.
     Escape,
 
+    /// The `F1` key.
     F1,
+    /// The `F2` key.
     F2,
+    /// The `F3` key.
     F3,
+    /// The `F4` key.
     F4,
+    /// The `F5` key.
     F5,
+    /// The `F6` key.
     F6,
+    /// The `F7` key.
     F7,
+    /// The `F8` key.
     F8,
+    /// The `F9` key.
     F9,
+    /// The `F10` key.
     F10,
+    /// The `F11` key.
     F11,
+    /// The `F12` key.
     F12,
+    /// The `F13` key.
     F13,
+    /// The `F14` key.
     F14,
+    /// The `F15` key.
     F15,
+    /// The `F16` key.
     F16,
+    /// The `F17` key.
     F17,
+    /// The `F18` key.
     F18,
+    /// The `F19` key.
     F19,
+    /// The `F20` key.
     F20,
+    /// The `F21` key.
     F21,
+    /// The `F22` key.
     F22,
+    /// The `F23` key.
     F23,
+    /// The `F24` key.
     F24,
 
-    /// Print Screen/SysRq.
+    /// The `Snapshot` / `Print Screen` key.
     Snapshot,
-    /// Scroll Lock.
+    /// The `Scroll` / `Scroll Lock` key.
     Scroll,
-    /// Pause/Break key, next to Scroll lock.
+    /// The `Pause` / `Break` key, next to the `Scroll` key.
     Pause,
 
-    /// `Insert`, next to Backspace.
+    /// The `Insert` key, next to the `Backspace` key.
     Insert,
+    /// The `Home` key.
     Home,
+    /// The `Delete` key.
     Delete,
+    /// The `End` key.
     End,
+    /// The `PageDown` key.
     PageDown,
+    /// The `PageUp` key.
     PageUp,
 
+    /// The `Left` / `Left Arrow` key.
     Left,
+    /// The `Up` / `Up Arrow` key.
     Up,
+    /// The `Right` / `Right Arrow` key.
     Right,
+    /// The `Down` / `Down Arrow` key.
     Down,
 
-    /// The Backspace key, right over Enter.
+    /// The `Back` / `Backspace` key, right over the `Enter` key.
```

Comment:
```suggestion
    /// The `Back` / `Backspace` key.
```
@alice-i-cecile I was actually talking about these. This one is actually incorrect for both my desktop and laptop keyboard. `\` sits in-between  `enter` and `backspace` for me. Here's an [image](https://tweakers.net/i/6_zaBrGnwQjzag8OAEZcX7LsUp8=/i/2002843126.jpeg) of my keyboard.

Reply:
Yeah, same here for me. I like this better.